src/rust/zh.manhuagui/src/helper.rs

```diff
@@ -0,0 +1,57 @@
+use alloc::{string::String, vec::Vec};
+
+pub fn i32_to_string(mut integer: i32) -> String {
```

Comment:
this doesn't need to be used anymore since you can just use `alloc::ToString` (search in the repo for it and you can see other sources using it, it just lets you do `.to_string()` on numbers)